stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -638,7 +638,7 @@ end
 
 function gemm_wrapper!(C::StridedVecOrMat{T}, tA::AbstractChar, tB::AbstractChar,
                        A::StridedVecOrMat{T}, B::StridedVecOrMat{T},
-                       _add = MulAddMul()) where {T<:BlasFloat}
+                       α::Number=true, β::Number=false) where {T<:BlasFloat}
```

Comment:
true and false? Why default values at all?

Reply:
I just decided to copy `gemv` for the default values. There was a default `_add` before, so I put in defaults for alpha and beta. Looks like it gets used in the 4-arg `gemm_wrapper()`.

Reply:
Oh, nevermind me. I actually didn't think `Bool <: Number`, but sure enough, it is.